storage/bucket.go

```diff
@@ -1562,7 +1554,12 @@ func toLifecycle(rl *raw.BucketLifecycle) Lifecycle {
 				NumNewerVersions:        rr.Condition.NumNewerVersions,
 			},
 		}
-		r.Condition.AgeInDays = getAgeCondition(rr.Condition.Age)
+		if rr.Condition.Age != nil {
+			r.Condition.AgeInDays = *rr.Condition.Age
+			if rr.Condition.Age == googleapi.Int64(0) {
```

Comment:
I would say `if *rr.Condition.Age == 0`, no need to create another pointer. (Actually I don't know if this works as written, since the pointers will not actually be equal.)